flink-java/src/test/java/org/apache/flink/api/java/operator/GroupingTest.java

```diff
@@ -127,6 +128,15 @@ public void testGroupByKeyFields5() {
 	}
 
 	@Test
+	public void testGroupByKeyFields6() {
```

Comment:
A more descriptive name or a comment would be helpful to understand the test case.